webapp/src/components/App/Singlestudy/PropertiesDialog/index.tsx

```diff
@@ -70,39 +71,90 @@ function PropertiesModal(props: Props) {
     return tpmTagsChanged;
   }, [initTags, tags]);
 
-  const onSubmit = async () => {
+  const publicModeList: Array<GenericInfo> = [
```

Comment:
Move it outside the component with the i18n keys only (or use the global i18n instance to translate them outside)